### PR TITLE
Outline segment group rendering

### PR DESCRIPTION
**Pull request by PaulHax**
image

### CODE REVIEW

**Comment by PaulHax** [comment]
Looks like this vtk.js commit broke label map rendering when the labelmap image is `Int16Array`

https://github.com/Kitware/vtk-js/commit/b04e59d1a10e5a6a06632799494a395ddcbfb2af#diff-18c5d83bebc14325d84e520000b95e06dadffa088fe27eef138af64744171b89R1891

**Comment by PaulHax** [comment]
@floryst This is good for review now.

File: src/components/vtk/VtkSegmentationSliceRepresentation.vue [hunk 1]
```diff
@@ -135,6 +138,35 @@ const applySegmentColoring = () => {
 
 watchEffect(applySegmentColoring);
 
+const configStore = useSegmentGroupConfigStore();
+const config = computed(() =>
+  configStore.getConfig(unref(viewId), unref(segmentationId))
+);
+
+const outlineThickness = computed(() => config.value?.outlineThickness ?? 2);
+// @ts-expect-error vtk.js types are incomplete
```
Comment (floryst) [inline src/components/vtk/VtkSegmentationSliceRepresentation.vue:147]: Can you please make a PR against vtk.js to fix these types?

**Review by floryst — APPROVED** [review]

**Comment by PaulHax** [comment]
> I'm wondering if we should move some of the opacity sliders to a menu. For instance, the segment opacity slider could be moved into the edit box.

Maybe a yes, and for that. Slicer segmentations opacity controls:

image

File: src/components/vtk/VtkSegmentationSliceRepresentation.vue (hunk 1 above)
Comment (PaulHax) [reply src/components/vtk/VtkSegmentationSliceRepresentation.vue:147]: https://github.com/Kitware/vtk-js/pull/3174

**Pull request merged**